README.md

```diff
@@ -1,3 +1,10 @@
+> [!CAUTION]
+> Dear Users,\
+> **This sample is deprecated and code has been archived on the 12th February 2024.**\
+> If you want to deploy an Erlang app on Scalingo, please use your own buildpack (this one can maybe serve as a starting point).\
```

Comment:
issue(blocking): this is not a buildpack so cannot be used as a starting point

Reply:
Woupsy copy/paste